apps/mtqueue/values-tucson-teststand.yaml

```diff
@@ -55,3 +55,16 @@ nfsMountpoint:
   readOnly: true
   server: nfs-obsenv.tu.lsst.org
   serverPath: /obs-env
+envSecrets:
+- name: AWS_ACCESS_KEY_ID
```

Comment:
Github is right: see envSecrets at line 14. Is that old entry obsolete? If you have time, please make this and the atqueue version look more similar, to ease future maintenance